build/configs/artik053/scripts/flash.ld

```diff
@@ -119,6 +119,7 @@ SECTIONS
 		*(COMMON)
```

Comment:
Please change the commit message
 "the following unused linekr variables" -> "the following unused linker variables"